README.md

```diff
@@ -3,289 +3,158 @@
 [![Build Status](https://travis-ci.org/Polymer/polymer-cli.svg?branch=master)](https://travis-ci.org/Polymer/polymer-cli)
 [![Build status](https://ci.appveyor.com/api/projects/status/3xc7rkapu39rw9fs/branch/master?svg=true)](https://ci.appveyor.com/project/justinfagnani/polymer-cli/branch/master)
 
-A command-line tool for Polymer projects.
+The command-line tool for Polymer projects and Web Components.
 
-For detailed help, check out the
-[Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli)
-guide.
+## Features
 
-## Overview
+  - **init** - Create a new Polymer project from pre-configured starter templates
```

Comment:
I want to be short and simple here so "built-in or third-party" is too wordy. I like "pre-configured" here but if that's a blocker for you I can remove.